.github/workflows/benchmark.yml

```diff
@@ -3,7 +3,7 @@
     
     on:
       push:
-        branches: [main]
+        branches: [main, dev/update-benchmark-script]
```

Comment:
is this still relevant

Reply:
nope, just added it and have removed it as well, was for local fork testing!